locale/pt-br/download/package-manager.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: page.hbs
+title: Instalando Node.js via gerenciador pacotes
+---
+
+# Instalando Node.js via gerenciador pacotes
+
+***Nota:*** Os pacotes nessa página são mantidos e suportados por seus respectivos empacotadores, **não** pela equipe principal do Node.js. Por favor, relate quaisquer problemas encontrados para o mantenedor do pacote.
```

Comment:
This is missing a sentence at the end:

> If it turns out your issue is a bug in Node.js itself, the maintainer will report the issue upstream.

Reply:
Não consigo pensar em uma boa tradução para "issue upstream"